.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: ci
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Cache Rust
+        uses: Swatinem/rust-cache@v1
+
+      - name: Test Rust
+        run: cargo test
```

Comment:
Is cargo present on the default image?

Reply:
This should work use the same setup in graphql tools https://github.com/dotansimha/graphql-tools-rs